compiler/passes/convert-uast.cpp

```diff
@@ -552,6 +725,16 @@ struct Converter {
     }
   }
 
+  Expr* visit(const uast::Tuple* node) {
+    CallExpr* actualList = new CallExpr(PRIM_ACTUALS_LIST);
```

Comment:
AFAIK `PRIM_ACTUAL_LIST` is really an artifact of parsing and there is no need to use it here. You could put a TODO comment if you don't want to change it in this PR though.